src/megatron/energon/flavors/crude.py

```diff
@@ -35,68 +26,34 @@ def __init__(
         *,
         subflavor: Optional[str] = None,
         subflavors: Optional[Dict[str, Any]] = None,
-        part_filter: Optional[Union[str, List[str], Callable[[str], bool]]] = None,
-        image_decode: Optional[ImageDecoder] = "torchrgb",
+        part_filter: Union[str, List[str], Callable[[str], bool]] = lambda _: True,
+        auto_decode: bool = True,
+        image_decode: ImageDecoder = "torchrgb",
```

Comment:
I realized that this setting does not only affect images, although called image_decode. Also affects e.g. numpy arrays and pickle data I think.

Reply:
Well, this is not true anymore. Now it should really only affect images?

Reply:
Ah ok cool 👍